src/main/resources/db/changelog/v.0.1/db.changelog-internship.yaml

```diff
@@ -18,13 +18,13 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: inship_start_date
-                  type: TIMESTAMP
+                  type: DATE
               - column:
                   name: inship_end_date
-                  type: TIMESTAMP
+                  type: DATE
               - column:
                   name: inship_publication_date
-                  type: TIMESTAMP
+                  type: DATE
```

Comment:
Did you notify other team members about modifying existing changesets?

Reply:
yes i have told them